core/import-csv.c

```diff
@@ -203,7 +203,7 @@ static int parse_dan_format(const char *filename, struct xml_params *params, str
 			for (i = 0; i < 5 && iter; ++i)
 				iter = strchr(iter + 1, '|');
 
-			if (iter && iter + 1) {
```

Comment:
and this one really should be
```
  if (iter && iter[1])
```

Reply:
Again, I don't see why. What's the difference between an unmatched '|' being at the end of the string or not?